run_cumulus_task.py

```diff
@@ -19,6 +19,19 @@ def set_sys_path():
     if os.path.isfile('cumulus-message-adapter.zip'):
         sys.path.insert(0, 'cumulus-message-adapter.zip')
 
+def handle_task_exception(
+    exception,
+    cumulus_message,
+    logger
+):
+    name = exception.args[0] if exception.args else None
```

Comment:
Previously, our exception handling was assuming the `exception.args[0]` is set, which may not be true for an exception such as `Exception()`. This change ensures our code can properly handle exceptions with no arguments